### ISSUE TITLE
Add button to download CSV of school applicants

### ISSUE THREAD

**Issue by sman591** [codeRIT/hackathon_manager#73]
I envision a few options:
* Basic info (name, email, acceptance status, school name)
* Basic + phone
* All data (only available to full-access admins)

**Comment by sman591** [comment]
@dquach217 Yeah - I think a dropdown menu with those three options would be perfect for it. If the user isn't a full-access admin, the third option is grayed out (`disabled` attr on the select option). Updated the description a bit

**Comment by dquach217** [comment]
@sman591 There exists an export dropdown for the format of csv and pdf. How do you want to proceed with this?

**Comment by sman591** [comment]
I think still having a dedicated export buttons/links would be the best here for simplicity's sake. However it might make sense to hook into the Datatables API instead of making a backend change, which could in theory be applied to _anywhere_ we have a table of data (school applicant list, bus list passengers, etc).

Right now this is manually achieved with the existing datatable by:
* Selecting "Show all rows"
* Toggling "column visibility" on for all possible columns
* Clicking "export" -> "CSV"

<img width="579" alt="Screenshot 2019-06-19 17 31 07" src="https://user-images.githubusercontent.com/1441807/59810087-72d96b80-92b8-11e9-9074-80e2a25aed89.png">

So, the JS would need to either emulate or somehow interact with the Datatables API to trigger a CSV download. Thoughts?
